config/environments/development.rb

```diff
@@ -50,53 +47,42 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  # Make template changes take effect immediately.
   config.action_mailer.perform_caching = false
 
-  # Print deprecation notices to the Rails logger.
-  config.active_support.deprecation = :log
+  # Configure the URL host for links
+  config.action_mailer.default_url_options = {
+    host: Credentials.fetch(:TEST_URL_HOST)
+  }
 
-  # Raise exceptions for disallowed deprecations.
-  config.active_support.disallowed_deprecation = :raise
+  Rails.application.routes.default_url_options[:host] = Credentials.fetch(:TEST_URL_HOST)
 
-  # Tell Active Support which deprecation messages to disallow.
-  config.active_support.disallowed_deprecation_warnings = []
```

Comment:
currently unused and was removed from the default config in https://github.com/rails/rails/commit/427fe4a6c5d259631b34ffcad2f9dff5231759e4